Tests/Common/Securities/CashTests.cs

```diff
@@ -677,7 +677,7 @@ private static TimeKeeper TimeKeeper
             new object[] { new BitfinexBrokerageModel(), Currencies.EUR, "XCHF", false, new[] { Symbol.Create("BTCXCHF", SecurityType.Crypto, Market.Bitfinex), Symbol.Create("BTCEUR", SecurityType.Crypto, Market.Bitfinex) } }, // No XCHFEUR, but indirect conversion exists
             new object[] { new BitfinexBrokerageModel(), Currencies.GBP, "XCHF", false, new[] { Symbol.Create("BTCXCHF", SecurityType.Crypto, Market.Bitfinex), Symbol.Create("BTCGBP", SecurityType.Crypto, Market.Bitfinex) } }, // No XCHFGBP, but indirect conversion exists
 
-            // *** GDAX ***
+            // *** Coinbase (deprecated: GDAX) ***
             // Trades USDC and USDT* (*Not yet trading live, but expected soon)
             // USDC Cases
             new object[] { new GDAXBrokerageModel(), Currencies.USD, "USDC", false, null }, // No USDCUSD, but does not throw! Conversion 1-1
```

Comment:
Would be good to change this to `CoinbaseBrokerageModel` and probably other references as well for more clarity since it's deprecated

Reply:
I added `[Obsolete("...")]` attribute in `GDAXBrokerageModel`
![image](https://github.com/QuantConnect/Lean/assets/45608740/dfa6b19e-d1c7-4947-9b3a-033b88697960)

Reply:
Do you mean change in all app ? to reduce amount of warning too ?

Reply:
Yeah, since you are deprecating it, you could replace its occurrences and so we internally have fewer warnings and get used to using the new Coinbase classes/naming